src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
Not that it matters right now, but let's rename the file to be `application.yaml` instead. It ends up being a bit nicer to read:

```
spring.data.mysql.username=4234
spring.data.mysql.password=hunter2
```

```yaml
spring.data.mysql:
  username: 4234
  password: hunter2
```